build/db2-migration-test-docker.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+###############################################################################
+# (C) Copyright IBM Corp. 2016, 2020
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################################
+set -ex
+
+echo "Preparing environment for fhir-server integration tests..."
+if [[ -z "${WORKSPACE}" ]]; then
+    echo "ERROR: WORKSPACE environment variable not set!"
+    exit 2
+fi
+
+# Set up installers and config files where docker processing can see them
+cd ${WORKSPACE}/fhir-install/docker
+./copy-dependencies-db2-migration.sh
+
+# Stand up a docker container running the fhir server configured for integration tests
+echo "Bringing down any fhir server containers that might already be running as a precaution"
+docker-compose kill
+docker-compose rm -f
+
+echo "Bringing up db2... be patient, this will take a minute"
+docker-compose build --pull db2
+docker-compose up -d db2
+echo ">>> Current time: " $(date)
+
+# TODO wait for it to be healthy instead of just Sleeping
+(docker-compose logs --timestamps --follow db2 & P=$! && sleep 100 && kill $P)
+
+echo "Deploying the Db2 schema..."
+./deploySchemaAndTenant.sh
+
+
+echo "Bringing up the FHIR server... be patient, this will take a minute"
+./copy-test-operations.sh
+docker-compose build --pull fhir
+docker-compose up -d fhir
+echo ">>> Current time: " $(date)
+
+echo "Migrating the schema to the latest version..."
+./copy-dependencies-db2.sh
+./updateSchema.sh
```

Comment:
can this script can always (and only) upgrade version 1 to the latest version?

Reply:
the `updateSchema` script is just invoking the `java -jar fhir-persistence-schema-*-cli.jar` `--update-schema` action with whatever version of that jar happens to be in `${SCHEMA}.
so, first we execute `copy-dependencies-db2.sh` to put the just built version there, then we run the script...the effect is that it will run update-schema with the "latest version" and invoke any migration steps from the code in the process.